src/toil/job.py

```diff
@@ -1718,6 +1728,20 @@ def checkpoint(self) -> bool:
         """Determine if the job is a checkpoint job or not."""
         return isinstance(self._description, CheckpointJobDescription)
 
+    @property
+    def files_to_use(self) -> Set[FileID]:
+        return self.description.files_to_use
+
+    @files_to_use.setter
+    def files_to_use(self, val: Set[FileID]):
+        self.description.files_to_use = val
+
+    def add_to_files_to_use(self, val: FileID):
+        self.description.files_to_use.add(val)
+
+    def remove_from_files_to_use(self, val: FileID):
+        self.description.files_to_use.remove(val)
```

Comment:
Since we have a getter and a setter and add/remove methods, we have 3 ways to add/remove files:
1. Use the getter and mutate the set
2. Use the getter, copy the set, change the copy, and use the setter
3. Use the dedicated add/remove methods

Really "There should be one-- and preferably only one --obvious way to do it."